source/math/FloatMatrix.ooc

```diff
@@ -45,6 +45,36 @@ FloatMatrix : cover {
 		result
 	}
 
+	setColumn: func (x: Int, values: ...) {
```

Comment:
Sure, I don't see why not. It'd be kinda like drawing images on a canvas. I think the loop should be changed so that we iterate over `x` in the inner loop, though (better for memory locality). It's likely going to be slower for vertical vectors (the inner loop might lead to a performance penalty when the width is 1).

We can still keep the functions that use a `FloatPoint3D`, for performance reasons.